src/test/java/edu/hm/hafner/analysis/parser/JavaDocParserTest.java

```diff
@@ -315,15 +315,10 @@ void issue55805() {
     @Test
     void shouldParseJavaDocsWarningsWithMavenSourcePlugin() {
         Report warnings = parse("issue63346.log");
-        assertThat(warnings).hasSize(3);
+        assertThat(warnings).hasSize(1);
```

Comment:
Why did you change that back? This parser is broken and should report 3 warnings:

```
[INFO] --- maven-javadoc-plugin:3.4.1:javadoc (default-cli) @ codingstyle ---
[INFO] Configuration changed, re-generating javadoc.
[WARNING] Javadoc Warnings
[WARNING] /Users/hafner/git/warnings-ng-plugin-devenv/codingstyle/src/main/java/edu/hm/hafner/util/FilteredLog.java:135: warning: no @return
[WARNING] public int size() {
[WARNING] ^
[WARNING] /Users/hafner/git/warnings-ng-plugin-devenv/codingstyle/src/main/java/edu/hm/hafner/util/LookaheadStream.java:49: warning: no comment
[WARNING] public String getFileName() {
[WARNING] ^
[WARNING] /Users/hafner/git/warnings-ng-plugin-devenv/codingstyle/src/main/java/edu/hm/hafner/util/ResourceExtractor.java:70: warning: no comment
[WARNING] public String getResourcePath() {
[WARNING] ^
[WARNING] 3 warnings
```